.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Reformatting of deploy-authdevs.sh
```

Comment:
Ohhh this is cool!!! Did not know you could do this, was going to ask you to squash that commit before I saw it was for changes not in this PR. ⭐

Reply:
:D the problem is, you need a second commit to update this file, as you need to know the SHA of the commit where the formatting changes were done! I wish there was some metadata you could put in the commit message like [reformat-only] :c